app/controllers/bestuurseenheid/reglementen/index.js

```diff
@@ -0,0 +1,10 @@
+import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+
+export default class ReglementenIndexController extends Controller {
+  @tracked page = 0;
+
+  get meetings() {
```

Comment:
these are not meetings, why are they called that
and in any case, you always have access to the route model in the template with `@model`, so this getter is a bit redundant (I'm not against it though! It can be nice to give a more descriptive name to the model, but if you do so, it should at least be correct...)

Reply:
yes I know that, I just wanted to be more descriptive and write that on my model and then get extracts from model. Sorry for naming it wrong